meshpy/geometry_set.py

```diff
@@ -218,8 +218,10 @@ def get_all_nodes(self):
         elif self.geometry_type is mpy.geo.line:
             nodes = []
             for element in self.geometry_objects[mpy.geo.line]:
-                nodes.extend(element.nodes)
-            return list(set(nodes))
+                for node in element.nodes:
+                    if node not in nodes:
+                        nodes.append(node)
+            return nodes
         else:
```

Comment:
Can we speed up these lines here?

Reply:
I have an idea how to make this a little bit more efficient, I will push these changes in the upcoming days.